ACRONYMS.md

```diff
@@ -0,0 +1,68 @@
+# Acronyms in Composable Kernel
+
+The following acronyms are used in the Composable Kernel codebase:
+
+| Acronym | Expansion | Explanation |
+|---------|-----------|-------------|
+| B2B     | Back-to-Back | Fused or sequential kernel operations (e.g., B2B GEMM) |
+| BF16    | Brain Floating Point 16 | 1 Signed bit, 8 Exponent bits, 7 Mantissa bits |
+| BF8     | 8-bit Brain Floating Point | 1 Signed bit, 3 Exponent bits, 4 Mantissa bits |
```

Comment:
Thanks! Nit, 'Sign', not 'Signed'. Also checked wiki and IEEE calls 'Significand' what I was calling 'Mantissa' so that should be also changed